src/configs/git_status.rs

```diff
@@ -10,6 +10,7 @@ pub struct GitStatusConfig<'a> {
     pub stashed: &'a str,
     pub ahead: &'a str,
     pub behind: &'a str,
+    pub uptodate: &'a str,
```

Comment:
I just noticed it might be better to bring this in line with the other names and call this `up_to_date`